examples/basics/ontologies.ipynb

```diff
@@ -46,7 +46,41 @@
         "!pip install labelbox"
```

Comment:
<p>Could you add "-q" to prevent the printing of all the installations</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1063/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>